src/main/java/edu/usc/ir/geo/gazetteer/GeoNameResolver.java

```diff
@@ -155,6 +178,27 @@ public GeoNameResolver(String indexPath) throws IOException {
 
 	}
 
+	public List<Location> searchNearby(Double latitude, Double longitude, Double distance, String indexerPath, int count) throws IOException {
```

Comment:
Naming a "distance" variable as-such without units leads to bugs in misunderstanding the units.  I suggest appending "Deg", or switch to "Km".